test/simplification/simplify/expected/array_root.json

```diff
@@ -0,0 +1,31 @@
+{
+  "originalSchema": {
+    "type": "array",
+    "items": [
+      {
+        "type": "string"
+      },
+      {
+        "type": "number"
+      }
+    ]
+  },
+  "type": "array",
+  "items": {
+    "type": [
+      "string",
+      "number"
+    ],
+    "originalSchema": {
+      "type": "array",
+      "items": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "number"
+        }
```

Comment:
This is not a true, because this schema indicates that you have an array of union, not a tuple. We should have schema like this: 

```js
"items": [
  { type: "string" },
  { type: "number" },
]
```

Reply:
Tuple will be integrated as feature in the future.